backends/advanced/src/advanced_omi_backend/routers/modules/queue_routes.py

```
@@ -626,4 +698,252 @@ async def clear_old_sessions(
+                logger.error(f"Error fetching streaming status: {e}")
+                return {"active_sessions": [], "stream_health": {}, "rq_queues": {}}
+
+        async def fetch_session_jobs(session_id: str):
+            """Fetch jobs for a specific session."""
+            try:
+                # Reuse the existing logic from get_jobs_by_session endpoint
+                from advanced_omi_backend.models.conversation import Conversation
+
+                # Get conversation IDs for this session
+                conversations = await Conversation.find(Conversation.audio_uuid == session_id).to_list()
+                conversation_ids = {conv.conversation_id for conv in conversations}
```

Ruff (0.14.1)

Local variable `conversation_ids` is assigned to but never used

Remove assignment to unused variable `conversation_ids`

(F841)